docs/competition/teams/photo-studio.md

```diff
@@ -0,0 +1,31 @@
+# Photo studio manager
+
+The role of the photo studio manager is to take nice photos of teams with their robots, and a photo of robot on their own.
+
+You should have:
+- A camera on a tripod
+- A table for the robot
+- A logbook to annotate what photos go to what team.
+- A photo backdrop
+
+Take many photos, and make sure the photos are in focus and correctly framed before dismissing the team.
+
+Take photos of:
+- The team posing with their robot
+- The robot itself
+
+Make sure you take note of which photos go to which team!
+
+Do this by doing both of the following:
+- Write down the ID of the photos for each team (for the first and last photos).
+- Take a photo of a piece of paper with the team's TLA before each set of team photos.
+
+# Photo studio helper
+
+The helper's job is to fetch teams to be photographed by the Photo studio manager. It's almost guaranteed that not all teams will want to be photographed immediately, so try to do it in order, but all teams must eventually be photographed!
```

Comment:
They will also be useful as a general photography assistant when not collecting teams.

Reply:
Fixed in 9e6504131f88a4eb91ae2fbc3c76d4fa2a41c4b1